### PR TITLE
Fix missing altcoins tab

### PR DESCRIPTION
**Pull request by Kukks**

### CODE REVIEW

File: BTCPayServer/Views/Shared/Bitcoin_Lightning_LikeMethodCheckout.cshtml [hunk 1]
```diff
@@ -151,8 +151,17 @@
         <div class="payment-tabs__tab" id="copy-tab" v-on:click="switchTab('copy')"  v-bind:class="{ 'active': currentTab == 'copy'}" >
             <span>{{$t("Copy")}}</span>
         </div>
-        
-        <div id="tabsSlider" class="payment-tabs__slider"  v-bind:class="['slide-'+currentTab]"></div>
+        @if (Model.CoinSwitchEnabled)
```
Comment (NicolasDorier) [inline BTCPayServer/Views/Shared/Bitcoin_Lightning_LikeMethodCheckout.cshtml:154]: I thought we removed CoinSwitch?

**Comment by Kukks** [comment]
No we removed changelly

**Comment by NicolasDorier** [comment]
Could be a plugin? :D

**Comment by Kukks** [comment]
> Could be a plugin? :D

For sure!

**Pull request merged**